bin/tests

```diff
@@ -23,5 +23,10 @@ if [ $# -eq 0 ]; then
 fi
 
 export REDIS_URL='redis:///'
-psql posthog -d postgres://posthog:posthog@localhost:5432 -c "drop database if exists test_posthog"
+
+PG_HOST="${PGHOST:=localhost}"
+PG_USER="${PGUSER:=posthog}"
+PG_PASSWORD="${PGPASSWORD:=posthog}"
+PG_PORT="${PGPORT:=5432}"
+psql posthog -d "postgres://${PG_USER}:${PG_PASSWORD}@${PG_HOST}:${PG_PORT}" -c "drop database if exists test_posthog"
```

Comment:
psql will use the env vars right? So there's no need to specify the connection string.

Reply:
Actually you'd want to remove the `_` in the env names for that do work

Reply:
It is only setting local variables not environment variables. So, it respects when someone has the env vars set. But someone who doesn't (me :) or a new contributor) gets sensible defaults

Reply:
better like this? https://github.com/PostHog/posthog/pull/7730/commits/4991f28808bf81cb5d4f9b21e41ddb6e773bbd21

Reply:
> It is only setting local variables not environment variables

Yeah maybe that's nicer actually 😳  I like your way.